README.md

```diff
@@ -25,11 +25,11 @@
 
 A web application supporting multiple user experience (UX) research methods.
 
-* [Card Sorting](https://en.wikipedia.org/wiki/Card_sorting)
-* [Tree Testing](https://en.wikipedia.org/wiki/Tree_testing)
-* [Product Reaction Cards](https://en.wikipedia.org/wiki/Microsoft_Reaction_Card_Method_%28Desirability_Testing%29)
-* [System Usability Scale (SUS)](https://en.wikipedia.org/wiki/System_usability_scale)
-* [Net Promoter Score (NPS)](https://www.netpromoter.com/know/)
+- [Card Sorting](https://en.wikipedia.org/wiki/Card_sorting)
```

Comment:
These have changed to - from *, Did we expect this to happen or is this linting shenanigans?

Reply:
Linting shenanigans. I'll revert